### PR TITLE
Logos functionality

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-react/src/ui/UFlow/Devices/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+interface LogoProps {
+  logo: string;
+  applicationName: string;
+}
+
+export const Logo = ({logo, applicationName}: LogoProps) => {
+  const alt = logo === 'none' ? logo : applicationName;
```
Comment (rzadp) [inline universal-login-react/src/ui/UFlow/Devices/Logo.tsx]: Why don't just `alt = applicationName` ?

**Comment by rzadp** [comment]
What if logo is `''` or `undefined`?

File: universal-login-react/src/ui/UFlow/Devices/Logo.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+interface LogoProps {
+  logo: string;
+  applicationName: string;
+}
+
+export const Logo = ({logo, applicationName}: LogoProps) => {
+  const src = logo === 'none'
+    ? 'https://universalloginsdk.readthedocs.io/en/latest/_images/logo.png'
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/UFlow/Devices/Logo.tsx]: make it constant for example `DEFAULT_LOGO` or `UNIVERSAL_LOGIN_LOGO`

File: universal-login-react/src/ui/UFlow/Devices/Logo.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+interface LogoProps {
+  logo: string;
+  applicationName: string;
+}
+
+const DEFAULT_LOGO = 'https://universalloginsdk.readthedocs.io/en/latest/_images/logo.png';
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/UFlow/Devices/Logo.tsx]: Move it to commons and use it in different locations like relayer configs.

Comment (JustynaBroniszewska) [reply universal-login-react/src/ui/UFlow/Devices/Logo.tsx]: rename `UNIVERSAL_LOGIN_LOGO`

File: universal-login-react/src/ui/UFlow/Devices/Logo.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+export const Logo = ({logo, applicationName}: LogoProps) => {
```
Comment (cehali) [inline universal-login-react/src/ui/UFlow/Devices/Logo.tsx]: Why not:
```js
export const Logo = ({logo = DEFAULT_LOGO, applicationName}: LogoProps) => {
```

File: universal-login-react/src/ui/UFlow/Devices/Logo.tsx (hunk 3 above)
Comment (JustynaBroniszewska) [reply universal-login-react/src/ui/UFlow/Devices/Logo.tsx]: `import {UNIVERSAL_LOGIN_LOGO as DEFAULT_LOGO}`

Comment (JustynaBroniszewska) [reply universal-login-react/src/ui/UFlow/Devices/Logo.tsx]: Lets do it in the next PR

**Pull request merged**